tests/app.spec.js

```diff
@@ -78,4 +80,25 @@ describe('api', () => {
       expect(rsp.status).toBe(404);
     });
   });
+
+  describe("MealFoods", () => {
+    test("Find foods associated with meals", async () => {
+      items = [
+        await Food.create({ name: "hashbrowns", calories: 300}),
+        await Food.create({ name: "bacon", calories: 500})
+      ]
+      return Meal.create({ name: "breakfast" }).then(async meal => {
+        console.log(meal)
+        await meal.addFoods(items)
+        await meal.getFoods().then(results => {
+          console.log(results)
+        expect(results[0].name).toBe("hashbrowns")
+        expect(results[0].calories).toBe(300)
+        expect(results[1].name).toBe("bacon")
+        expect(results[1].calories).toBe(500)
+        expect(results[1].name).not.toBe("potato chips")
+        })
+      })
```

Comment:
```suggestion
      let meal = await Meal.create({name: 'breakfast'});
      await meal.addFoods(items);
      let results = await meal.getFoods();
      expect(results[0].name).toBe("hashbrowns")
      expect(results[0].calories).toBe(300)
      expect(results[1].name).toBe("bacon")
      expect(results[1].calories).toBe(500)
      expect(results[1].name).not.toBe("potato chips")
```

Reply:
... I don't think multiline comments work yet. That change is supposed to be lines 90 to 101 not 101 to 108... I'd recommend not hitting the 'commit suggestion' button and just doing it in atom.

Reply:
I made the requested change in atom and pushed the commit below: